invenio_communities/templates/semantic-ui/invenio_communities/details/about/index.html

```diff
@@ -8,13 +8,13 @@
 #}
 
 {% extends "invenio_communities/details/base.html" %}
+{% from "invenio_communities/details/macros/custom_fields.html" import list_vocabulary_values, list_string_values, show_custom_field_depending_on_type %}
```

Comment:
not a fan of long names, what about 
```suggestion
{% from "invenio_communities/details/macros/custom_fields.html" import list_vocabulary_values, list_string_values, show_custom_field %}
```

In the end what kind of logic is inside, is abstracted.

Reply:
Updated! :)